### ISSUE TITLE
How to try it?

### ISSUE THREAD

**Issue by cdecompilador** [pwouik/rust_voxel_engine#1]
When I run it everything is sky, why?

**Comment by pwouik** [comment]
create save/region/ directory

**Comment by cdecompilador** [comment]
something definitely changed, but still screen all blue

**Comment by pwouik** [comment]
I was testing stuff is the last commit
replace chunk_renderer.rs L611 with `load: wgpu::LoadOp::Clear(1.0),`

**Comment by cdecompilador** [comment]
nope, still nothing, I'll wait for the next commit to try

**Comment by pwouik** [comment]
yeah idk it work for me with these step, maybe you need renderdoc installed
or you can try older commits
